modin/pandas/test/dataframe/test_binary.py

```diff
@@ -360,7 +360,7 @@ def test_non_commutative_multiply_pandas():
     # The non commutative integer class implementation is tricky. Check that
     # multiplying such an integer with a pandas dataframe is really not
     # commutative.
-    pandas_df = pd.DataFrame([[1]], dtype=int)
+    pandas_df = pandas.DataFrame([[1]], dtype=int)
```

Comment:
Why do we need to test pandas? Doesn't pandas itself do this?

Reply:
See the comment just above. Getting the non-commutative multiply class right was tricky. Before we check that the multiply with Modin matches the multiply with pandas, we should check that the multiply with pandas is really non-commutative. If it's not, there's no point in running `test_non_commutative_multiply`.